CMakeLists.txt

```diff
@@ -17,10 +17,10 @@ project(TurboMind LANGUAGES CXX CUDA)
 
 find_package(CUDA 10.2 REQUIRED)
 
-# if(${CUDA_VERSION_MAJOR} VERSION_GREATER_EQUAL "11")
-#   add_definitions("-DENABLE_BF16")
-#   message("CUDA_VERSION ${CUDA_VERSION_MAJOR}.${CUDA_VERSION_MINOR} is greater or equal than 11.0, enable -DENABLE_BF16 flag")
-# endif()
+if(${CUDA_VERSION_MAJOR} VERSION_GREATER_EQUAL "11")
```

Comment:
Does V100 support BF16?

Reply:
Nope

Reply:
so, if building turbomind on v100 with cuda 11.8, what will happen

Reply:
and if the prebuilt package includes BF16, what will happen if users use it on v100 platform

Reply:
It should be OK to bulid/run on device without bf16 support.
Bf16 inference on such device will leads to runtime error.